examples/sas_logical_interconnect.rb

```diff
@@ -64,10 +64,6 @@
   client my_client
   firmware 'ROM Flash - SPP'
   firmware_data(
```

Comment:
I changed the attributes because these attributes do not work with sas logical connection.